examples/preprocessing/plot_all_scaling.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+=============================================================
+Compare the effect of different scalers on data with outliers
+=============================================================
+
+The feature 0 and feature 5 of california housing dataset contains large
+outliers that can make visualization of the data difficult.
+
+Also linear models like :class:`sklearn.linear_model.SVM` require data which is
+normalized to the range [-1, 1].
```

Comment:
approximately normalized to the [-1, 1] or [0, 1] range, or at the very least have all the features on the same scale.